apps/src/code-studio/components/progress/ResourcesDropdown.story.jsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ResourcesDropdown from './ResourcesDropdown';
-import {allowConsoleWarnings} from '../../../../test/util/testUtils';
```

Comment:
It's so interesting that this was a thing and I'm not sure I understand how it works within storybook. That said, it looks like you're removing it so maybe it doesn't matter!

Reply:
@madelynkasula - I'm just realizing that this might not be okay to remove. Do you happen to know how this historically appeared in the storybook (so I can check to see how it works in the new one)?

Reply:
if unit tests pass without it, you don't need to add it back. we have some hooks on unit tests that make it so that the build fails if any tests log warnings/errors to the console

Reply:
Nice! Thank you!